bot_cli/commands.py

```diff
@@ -264,3 +263,43 @@ def birthdays(args):
         print_success(result)
     else:
         print_warn(f"There is no one to celebrate birthday for next {period} day(s)")
+
+
+@contact_not_found_error
+@add_address_error
+def add_address(args):
+    try:
+        name, *address = args
+        name = Name(name)
+        address = " ".join(address)
+    except:
+        raise CommandError
+
+    if address_book.find(name):
+        record: Record = address_book.find(name)
+        record.add_address(Address(address))
+    else:
+        raise ContactNotFoundError
+
+    address_book.save_contacts(FILE_PATH)
+    print_success("Address added successfully")
+
+
+@contact_not_found_error
+@show_address_error
+def show_address(args):
+    try:
+        name = args[0]
+        name = Name(name)
+    except:
+        raise CommandError
+
+    if address_book.find(name):
```

Comment:
The same as above - separate variable for the found contact

Reply:
Then you can have a better flow like this:
```python
record = address_book.find(name)
if not record:
    raise ContactNotFoundError
...
```

Reply:
Updated